app/controllers/concerns/api_actionable.rb

```diff
@@ -105,4 +106,11 @@ def load_api_actions_from_api_resource
     @redirect_action = @api_resource.send(api_action_name).where(action_type: 'redirect').reorder(:created_at).last if @redirect_action.present?
     @serve_file_action = @api_resource.send(api_action_name).where(action_type: 'serve_file').reorder(:created_at).last if @serve_file_action.present?
   end
+
+  def track_create_event
+    ahoy.track(
+      "api-resource-create",
+      { visit_id: current_visit.id, api_resource_id: @api_resource.id, user_id: current_user&.id }
```

Comment:
@Pralish could be useful to tag the API Namespace ID here as well

Reply:
It's done. Thanks for the feedback